python/metatensor-torch/tests/atomistic/openmm_force.py

```diff
@@ -0,0 +1,186 @@
+import os
+
+import ase.io
+import metatensor_lj_test
+import numpy as np
+import pytest
+from ase.build import bulk
+
+from metatensor.torch.atomistic.ase_calculator import MetatensorCalculator
+from metatensor.torch.atomistic.openmm_force import get_metatensor_force
+
+
+try:
+    import NNPOps  # noqa: F401
+    import openmm
+    import openmmtorch  # noqa: F401
+
+    HAS_OPENMM = True
+except ImportError:
+    HAS_OPENMM = False
+
+CUTOFF = 5.0
+SIGMA = 1.5808
+EPSILON = 0.1729
+
+
+def model():
+    return metatensor_lj_test.lennard_jones_model(
+        atomic_type=29,
+        cutoff=CUTOFF,
+        sigma=SIGMA,
+        epsilon=EPSILON,
+        length_unit="Angstrom",
+        energy_unit="eV",
+        with_extension=False,
+    )
+
+
+def model_different_units():
```

Comment:
this might not be so interesting to run here, I did it in ASE to ensure our code did unit conversions right, but we don't have to do it for all interfaces.